acmwebsite/templates/schedule.xhtml

```diff
@@ -10,6 +10,7 @@
 
   <body py:block="body" py:strip="True">
     <h1 class="page-header">Upcoming Meetings</h1>
+    <p>Subscribe to the <a href="${tg.url('/schedule/acm.ics')}">iCal calendar</a>.</p>
```

Comment:
Should be at bottom of page, or in a sidebar thing?

Reply:
![2017-09-24-14 33 45](https://user-images.githubusercontent.com/16734772/30786584-6602b4c8-a135-11e7-8e59-b3ca75fc37f9.png)

Something like this?

Reply:
Looks a little bit silly without anything else in the sidebar. OK if we use for list of scheduled reminders when we get that feature working.

Also needs instructions on how to add to Google Calendar (link to external site that describes this well if there is one).

Reply:
I think sidebar is good. When we add the scheduled reminders feature, it will be merely additive, rather than a re-arrangement of the UI.

Also, I was looking into if it was possible to include a link to "Add to Google Calendar". If I can't figure that out, I'll add a link to instructions.